### DISCUSSION TITLE
cachedir file load

### DISCUSSION THREAD

**Discussion by ruanshengbin** [bytedeco/javacpp#569] [Q&A] [answered]
can config  cache  exists  file  not cover it

**Comment by saudet** [comment]
I think the answer is "yes"...

**Comment by ruanshengbin** [comment, marked as answer]
why do not use file md5 to check file

**Reply by saudet** [reply]
Because it would make loading even slower than it is now. It currently only checks the sizes and dates of files, which does not slow down loading. I've never encountered a situation where this wasn't sufficient. However, we can also add a checksum as an option. Please feel free to open a pull request to that effect: https://github.com/bytedeco/javacpp/pulls. Contributions are always welcome!
